clarity/contracts/dcards-nft.clar

```diff
@@ -164,5 +162,5 @@
 
 ;; check status on last mint
 (define-read-only (mint-check)
-  (print {evt: "mint-check", last: (var-get last-id), owner: tx-sender, amount: (var-get mint-price)})
+  (print {evt: "mint-check", last: (var-get last-id), owner: (get-owner (var-get last-id)), amount: (var-get mint-price)})
```

Comment:
will later update to return a value for this, just printing is not very useful. 
it could also just return the `mint-info` function for last-id...